### ISSUE TITLE
Add global filters to Vue

### ISSUE THREAD

**Issue by Pacheco95** [nuxt/typescript#477]
I have created a plugin to add global Vue filters but when I'm using them in double mustache expressions intellisense is not working

```ts
import Vue from 'vue'

Vue.filter('capitalized', (s: string) => s[0].toUpperCase() + s.substring(1))
```

Using local filters with `vue-property-decorator` works as expected
```ts
@Component({
  filters: {
    capitalized: (s: string) => s[0].toUpperCase() + s.substring(1),
  },
})
```

How can I get intellisense to work with global filter definitions?

**Comment by kevinmarrec** [comment]
@Pacheco95 I'm not sure where intellisense should trigger, could you show me a screen with the intellisense working as expected when using `vue-property-decorator` ?

**Comment by Pacheco95** [comment]
@kevinmarrec sure!
![image](https://user-images.githubusercontent.com/12385467/104920040-f61be100-5975-11eb-863d-1e3b8fed217d.png)

**Comment by kevinmarrec** [comment]
Alright I see, well first the intellisense here in templates are provided by your editor and not by this package.
And when setuping global filters, your editor just can't know you have register it globally to provide you intellisense on it.

I'm closing cause it's not related to `nuxt/typescript` itself.

I invite you to check https://stackoverflow.com/questions/56601270/how-do-i-make-typescript-recognize-my-custom-global-filter , I think you need to extend Vue types to make it work.

Finally, you may find help on Vue Discord : https://t.co/Nfb5gWGlfU?amp=1
